libs/challenge-registry/challenge-search/src/lib/challenge-search.component.scss

```diff
@@ -64,7 +64,7 @@
 .facets {
   margin: 18px;
   padding: 48px 21px;
-  min-width: 320px;
+  max-width: 320px;
```

Comment:
Hm, I originally changed it because the facets sidebar was really wide and took over half of the screen.  But looks like this is no longer the case, I'm assuming due to changes done elsewhere.

I'll revert for now.